### PR TITLE
Move qubes-rpc installation from the root Makefile to qubes-rpc Makefile

### PR DESCRIPTION
**Pull request by aidecoe**
qubes-rpc has its own Makefile that's responsible for building some executables. The root Makefile was installing qubes-rpc files. To make qubes-rpc a bit more indepdent from core-agent root Makefile and to ease potential maintainer work on packaging qubes-rpc separately, the installation has been moved to qubes-rpc Makefile. Moreover that should make the Makefiles easier to read and maintain.

Use built-in rules in qubes-rpc makefile. That allows a build system to customize compiler and linker and pass extra flags to these. Remove `-g` as default flag and enable it only when `DEBUG` variable is set.

### CODE REVIEW

File: qubes-rpc/Makefile [hunk 1]
```diff
@@ -1,16 +1,75 @@
...
+BINDIR ?= /usr/bin
+LIBDIR ?= /usr/lib
+SYSCONFDIR ?= /etc
+QUBESLIBDIR = $(LIBDIR)/qubes
+QUBESRPCCMDDIR = $(SYSCONFDIR)/qubes-rpc
+QUBESCONFDIR = $(SYSCONFDIR)/qubes
+QUBESRPCCONFDIR = $(QUBESCONFDIR)/rpc-config
+
+CPPFLAGS := -I.
+CFLAGS := $(if $(DEBUG),-g,) -O2 -Wall -Wextra -Werror -fPIC -pie $(CFLAGS)
```
Comment (marmarek) [inline qubes-rpc/Makefile]: I'd prefer to keep `-g` enabled. Most packaging systems require it, to extract debug symbols into separate files/packages.

File: qubes-rpc/thunar/Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+SYSCONFDIR ?= /etc
+LIBDIR ?= /usr/lib
+QUBESLIBDIR = $(LIBDIR)/qubes
+
+.PHONY: install
+
+install:
+	install -D -t $(DESTDIR)$(QUBESLIBDIR) qvm-actions.sh
```
Comment (marmarek) [inline qubes-rpc/thunar/Makefile]: You forgot to move this file (see travis).

Comment (aidecoe) [reply qubes-rpc/thunar/Makefile]: Fix this and 1 more file. I have forgotten to test thunar installation. Tested now.

File: qubes-rpc/Makefile (hunk 1 above)
Comment (aidecoe) [reply qubes-rpc/Makefile]: Ok, I kept it enabled by default and disabled only if DEBUG=0 is passed.

File: qubes-rpc/Makefile [hunk 3]
```diff
@@ -1,16 +1,78 @@
...
+		qubes.InstallUpdatesGUI \
+		qubes.ResizeDisk \
+		qubes.StartApp \
+		qubes.PostInstall \
+		qubes.GetDate \
+		qubes.ShowInTerminal \
+		qubes.ConnectTCP
+	install -d $(DESTDIR)/etc/qubes/rpc-config
+	for config in *.config; do \
+		install -D -m 0644 "$$config" "$(DESTDIR)$(QUBESRPCCONFDIR)/$${config#.config}"; \
```
Comment (marmarek) [inline qubes-rpc/Makefile]:
```suggestion
		install -D -m 0644 "$$config" "$(DESTDIR)$(QUBESRPCCONFDIR)/$${config%.config}"; \
```

**Comment by aidecoe** [comment]
This time only DISTS_VM=centos7 failed, but it's not as clear why. Could you help, please? @marmarek

**Comment by marmarek** [comment]
Missing dependency for centos fixed and not it fails for a different reason - directory `etc/qubs-suspend-pre.d` not created.

**Comment by marmarek** [comment]
There are some more places where `install -D` is broken on centos (see travis).

**Comment by marmarek** [comment]
```
+ make -C qubes-rpc/kde DESTDIR=/home/user/rpmbuild/BUILDROOT/qubes-core-agent-4.1.5-1.centos7.x86_64 install

make: Entering directory `/home/user/rpmbuild/BUILD/qubes-core-agent-4.1.5/qubes-rpc/kde'

install -D -t /home/user/rpmbuild/BUILDROOT/qubes-core-agent-4.1.5-1.centos7.x86_64/usr/share/kde4/services -m 0644 *.desktop

install: failed to access '/home/user/rpmbuild/BUILDROOT/qubes-core-agent-4.1.5-1.centos7.x86_64/usr/share/kde4/services': No such file or directory
```

**Pull request merged**